db/schema.rb

```diff
@@ -123,6 +129,23 @@
     t.index ["wine_id"], name: "index_reviews_on_wine_id", using: :btree
   end
 
+  create_table "sodas", force: :cascade do |t|
+    t.integer  "user_id"
+    t.integer  "country_id"
+    t.string   "name"
+    t.string   "brand"
+    t.string   "origin_data"
+    t.boolean  "is_diet"
+    t.boolean  "is_caffeinated"
+    t.string   "color"
+    t.integer  "users_id"
```

Comment:
yes, this was an error from the bad .references usage earlier. Should be fixed in next pr